xlators/storage/posix/src/posix-mem-types.h

```diff
@@ -21,6 +21,7 @@ enum gf_posix_mem_types_ {
     gf_posix_mt_inode_ctx_t,
     gf_posix_mt_mdata_attr,
     gf_posix_mt_uring_ctx,
+    gf_posix_diskxl_t,
```

Comment:
For consistency with the other names you should add `_mt_`.

```suggestion
    gf_posix_mt_diskxl_t,
```